README.md

```diff
@@ -21,8 +21,8 @@ It is intended for use with editor/IDE integration.
 
 ## Supported linters
 
-- [go vet](http://godoc.org/golang.org/x/tools/cmd/vet/) - Reports potential errors that otherwise compile.
-- [gotype](golang.org/x/tools/cmd/gotype) - Syntactic and semantic analysis similar to the Go compiler.
+- [go vet](https:///golang.org/x/tools/cmd/vet/) - Reports potential errors that otherwise compile.
```

Comment:
Also, this was linked to the godoc because the golang.org URLs just redirect there...seems a bit redundant to me.